app.py

```diff
@@ -9,10 +9,12 @@
 from wtforms.validators import DataRequired, URL
 from flask_ckeditor import CKEditor, CKEditorField
 from datetime import date
-from camera import VideoCamera
+# from camera import VideoCamera
```

Comment:
Why did you commented this here?

Reply:
Extremely sorry for that , i have corrected it can you please review it now.

Reply:
![Screenshot (41)](https://user-images.githubusercontent.com/56766678/151839130-b5308eb4-37e6-4d0c-b972-1152e8a8d41d.png)
i am getting this error in camera.py module

Reply:
Extremely sorry for the inconvenience, i have submitted new PR ,can you please review it again.